src/Aacotroneo/Saml2/Saml2Auth.php

```diff
@@ -84,6 +85,11 @@ function acs()
             return $errors;
         }
 
+        Session::put('federationData', [
+            'sessionIndex' => $auth->getSessionIndex(),
+            'nameId' => $auth->getNameId()
+        ]);
```

Comment:
This `auth` is passed to the User (which is latter exposed to the project). You could get this information from there (extend the user to make it available)